modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-partner-portal/osb-site-initializer-partner-portal-test/src/testFunctional/tests/PRMDashboards.testcase

```diff
@@ -95,6 +95,81 @@ definition {
 		}
 	}
 
+	@description = "This is a test for LPS-179357. Verify that the value of Approved Claims is displayed on MDF Dashboard"
+	@priority = 4
+	test ClaimApprovedValueAppearInDashboard {
+		var budgetValue = 2000;
+
+		task ("Given I get the current amount of approved MDF Claim Value") {
+			var initialValue = PRMDashboards.getLegendValue(
+				chartName = "Request",
+				legendName = "Claim Approved");
+		}
+
+		task ("And I create a new MDF Request") {
+			JSONPRM.createMDFRequest(
+				accountEntryName = "Company Name 1",
+				budgetValueList = ${budgetValue},
+				campaignName = "Campaign Name 1");
+		}
+
+		task ("And I go back to MDF listing page") {
+			PRMNavigator.openSitePage(pageName = "MDF Requests");
+		}
+
+		task ("And I go to approve the MDF") {
+			PRMMDFRequest.goToMDF(campaignName = "Campaign Name 1");
+
+			PRMMDFRequest.changeMDFStatus(statusButton = "Approve");
+		}
+
+		task ("And I create a new Claim") {
+			PRMMDFClaim.createNewClaim(
+				listLeads = "Document_1.xlsx",
+				reimbursementInvoice = "Document_1.png",
+				thirdInvoice = "Document_1.png");
+		}
+
+		task ("And I go to MDF Claim listing page") {
+			PRMNavigator.openSitePage(pageName = "MDF Claim");
+		}
+
+		task ("And I get the Claim ID") {
+			var request_numID = PRMMDFClaim.getClaimRequestID();
+		}
+
+		task ("And I search for Claim ID") {
+			PRMMDFClaim.searchClaim(claimID = ${request_numID});
+		}
+
+		task ("And I go to approve the MDF Claim") {
+			PRMMDFClaim.gotoMDFClaim(goClaim = 1);
+
+			PRMMDFRequest.changeMDFStatus(statusButton = "Approve");
+		}
+
+		task ("When I go back to Dashboard page and get the new amount of approved MDF Claims approved Value") {
```

Comment:
Hi @marinaasant0s, there is a typo. Please rename your task to 'When I go back to Dashboard page and get the new amount of approved MDF Claims Value'